README.md

```diff
@@ -260,10 +275,8 @@ are constructed using shared metadata, which lives in the [problem-specification
 
 ## Contributing Guide
 
-For an in-depth discussion of how exercism language tracks and exercises work,
-please see the
-[contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data)
-
+For an in-depth discussion of how exercism language tracks and exercises work, please see the
+[contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).  If you're just getting started and looking for an easy way to get involved, take a look at  [regenerating the test suites](#regenerating-a-test-suite), [porting an exercise from another language](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md), or [creating an automated test generator](#implementing-a-generator).
```

Comment:
Add your new section as a new paragraph rather than continuing the line.

Reply:
I'm not sure these things are easy ways to get involved.
They are things that are helpful, but can get quite complicated.